plugin/helpers_bots.go

```diff
@@ -57,6 +129,10 @@ func (p *HelpersImpl) EnsureBot(bot *model.Bot) (retBotId string, retErr error)
 		} else {
 			p.API.LogError("Plugin attempted to use an account that already exists. Convert user to a bot account in the CLI by running 'mattermost user convert <username> --bot'. If the user is an existing user account you want to preserve, change its username and restart the Mattermost server, after which the plugin will create a bot account with that name. For more information about bot accounts, see https://mattermost.com/pl/default-bot-accounts", "username", bot.Username, "user_id", user.Id)
 		}
+		setImagesErr := p.setBotIconAndProfileImage(user.Id, o.ProfileImagePath, o.IconImagePath)
```

Comment:
i wonder if it makes sense to call this if this the user is not a bot. I think we should return an error if this user is not a bot. Not sure why we didn't in the first place 🤔

Reply:
There is a test for this functionality too: `should return the non-bot account but log a message if user exists with the same name and is not a bot`. So should I return an error and update this test? Also, should I still keep the log?

Reply:
ah i see. Errrmm. maybe that is intended. Alright maybe lets leave it the way it is then. My apologies. Not sure why we would return a user as a bot even though it's not a bot 🤔